gui/layers/_image_layer.py

```diff
@@ -270,3 +272,45 @@ def interpolation_functions(self):
 
     def _clim_range_default(self):
         return [np.min(self.image), np.max(self.image)]
+
+    def get_value(self, position, indices):
+        """Returns coordinates, values, and a string
+        for a given mouse position and set of indices.
+
+        Parameters
+        ----------
+        position : sequence of two int
+            Position of mouse cursor in canvas.
+        indices : sequence of int or slice
+            Indices that make up the slice.
+
+        Returns
+        ----------
+        coord : sequence of int
+            Position of mouse cursor in data.
+        value : int or float or sequence of int or float
+            Value of the data at the coord.
+        msg : string
+            String containing a message that can be used as
+            a status update.
+        """
+        transform = self._node.canvas.scene.node_transform(self._node)
+        pos = transform.map(position)
+        pos = [clip(pos[1],0,self.shape[0]-1), clip(pos[0],0,self.shape[1]-1)]
+        coord = copy(indices)
+        coord[0] = int(pos[0])
+        coord[1] = int(pos[1])
+        value = self._slice_image(coord)
+        msg = f'{coord}'
+        msg = msg + ', %s' % self.name + ', value '
```

Comment:
I think this would be cleaner as `msg = f'{coord}, {self.name}, {value}'`

Reply:
improved